bindings/lua/lua-opendal-0.1.0-1.rockspec

```diff
@@ -0,0 +1,27 @@
+package = "lua-opendal"
+version = "0.1.0-1"
```

Comment:
I don't know which version number we can choose here, What do you think in here? @Xuanwo